app/views/menus/show.html.haml

```diff
@@ -0,0 +1,4 @@
+%h2 Menu
+
+=@restaurant.name
+=@restaurant.menu.name
```

Comment:
I see that you set a variable in the controller called `@menu` but you are not using it in the template. Can you see the mistake here?